src/index.html

```diff
@@ -1,22 +1,438 @@
+
+                <p class="detail__description">
+                  The removable battery has up to 70km battery autonomy and
+                  weighs only 2.4 kg. Lights integrated into the frame give you
+                  always-on visibility day and night.
+                </p>
+              </article>
+
+              <article class="detail">
+                <div class="detail__photos">
+                  <a
+                    href="#"
+                    class="detail__link detail__link--square"
+                  >
+                    <img
+                      src="./images/Hydraulic disc brakes & Lightweight - square.png"
+                      alt="Auto Unlock"
+                      class="detail__photo"
+                    />
+                  </a>
+
+                  <a
+                    href="#"
+                    class="detail__link detail__link--wide"
+                  >
+                    <img
+                      src="./images/Hydraulic disc brakes & Lightweight - wide.png"
+                      alt="Auto Unlock"
+                      class="detail__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="detail__title">
+                  Hydraulic disc brakes & Lightweight
+                </h3>
+
+                <p class="detail__description">
+                  Brakes with total stopping power the second you make contact.
+                  The removable battery has up to 70km battery autonomy and
+                  weighs only 2.4 kg. Lights integrated into the frame give you
+                  always-on visibility day and night.
+                </p>
+              </article>
+
+              <div class="button details__button">
+                <a
+                  href="#"
+                  class="button__link"
+                >
+                  Explore
+                </a>
+              </div>
+            </div>
+          </section>
+
+          <section
+            class="contacts"
+            id="contacts"
+          >
+            <h2
+              class="contacts__title section-title section-title--on-desktop-font"
+            >
+              Contact us
+            </h2>
+            <div class="contacts__wrapper">
+              <div class="contacts__form">
+                <form
+                  onsubmit="this.reset(); return false;"
+                  class="form"
+                  action="/"
+                  method="post"
+                  name="message-form"
+                >
+                  <input
+                    class="form__field"
+                    type="text"
+                    name="name"
+                    placeholder="Name"
+                    required
+                  />
+
+                  <input
+                    class="form__field"
+                    type="email"
+                    name="email"
+                    placeholder="Email"
```

Comment:
Placeholder should be an example of a valid value for the field [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Placeholders%20should%20give%20examples%20of%20what%20should%20be%20added%2C%20to%20make%20the%20expected%20format%20clear)